packages/meditrak-server/src/routes/index.js

```diff
@@ -149,6 +153,7 @@ export default {
   getDisasters: useRouteHandler(GETDisasters),
   getDashboardReports: useRouteHandler(GETDashboardReports),
   getDashboardGroups: useRouteHandler(GETDashboardGroups),
+  getIndicators: useRouteHandler(GETIndicators),
```

Comment:
Could make a `BESAdminGetHandler` for consistency, even if it's the only one to use it

Reply:
Apparently I already wrote one but forgot about it